### PR TITLE
Add "onyx-server on AWS" tutorial

### PR DESCRIPTION
**Pull request by nietaki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,96 @@ The `DEBUG` environment variable can be used to activate logs, ex:
...
+| ALL Traffic | ALL      | ALL        | 0.0.0.0/0 | ALL Traffic |
+
+#### 2. Create a subnet for your Onyx Server
+
+Go to [subnet management](https://eu-west-1.console.aws.amazon.com/vpc/home?region=eu-west-1#subnets:)
+in the AWS dashboard and create a new subnet in your VPC. Make sure it's within
+the vpc CIDR range. For example if the VPC CIDR is `10.0.0.0/16`, the sg
+IPv4 CIDR block can be `10.0.0.1/24`.
+
+### 3. Create an SSH key
```
Comment (mosic) [inline README.md]: `s/###/####`

File: README.md [hunk 2]
```diff
@@ -34,6 +34,96 @@ The `DEBUG` environment variable can be used to activate logs, ex:
...
+You're going to use it to connect to the Onyx Server EC2 node.
+
+```bash
+$ mkdir ~/ssh
+$ aws ec2 create-key-pair --key-name my_key --output text --query KeyMaterial > ~/ssh/my_key.pem
```
Comment (mosic) [inline README.md:89]: Permissions need to be properly set too:
`$ chmod 400 ~/ssh/my_key.pem`

File: README.md [hunk 3]
```diff
@@ -34,6 +34,96 @@ The `DEBUG` environment variable can be used to activate logs, ex:
...
 
+### Setting up an Onyx Server on AWS
+
+You can use a pre-built [AMI](https://en.wikipedia.org/wiki/Amazon_Machine_Image)
+to conveniently set up an onyx-server on AWS.
+
+To do it, make sure you have an AWS account and your AWS CLI is configured to 
+use the `eu-west-1` (Ireland) region as default. We're going to assume you have a
+[VPC](https://eu-west-1.console.aws.amazon.com/vpc/home?region=eu-west-1#)
+configured (created and connected to an Internet Gateway) in this region.
```
Comment (mosic) [inline README.md]: Aside from a VPC and an Internet Gateway, Route Table has to be configured, with a route set like this:

| Destination        | Target                                                |  Status   | Propagated |
| ---                      | ----                                                      | ---          | ---                |
| 10.0.0.0/16        | local                                                   |  Active   | No               |
| 0.0.0.0/0            | \<INTERNET GATEWAY ID HERE> |  Active   | No               |

**Review by mosic — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (nietaki) [reply README.md]: That's kind of what I meant in `created and connected to an Internet Gateway`. I'll add a link to the Route Tables there to make it more explicit what needs to happen.

**Review by cayblood — APPROVED** [review]
Looks good! 
twin peaks

**Pull request merged**